app/models/decorators/mvi_user_decorator.rb

```diff
@@ -12,6 +14,23 @@ def create
     raise Common::Exceptions::ValidationErrors, @user unless @user.valid?
     message = create_message
     response = @mvi_service.find_candidate(message)
+    user_with_mvi_attributes(response)
+  rescue MVI::RecordNotFound
+    # TODO(AJD): add metric
+    Rails.logger.error "MVI record not found for user: #{@user.uuid}"
+    @user.attributes = { mvi: NOT_FOUND }
```

Comment:
Thought that too the first I saw it, virtus gem implements `#attributes=` as mass assignment and does not affect other attrs.